src/Opg/Lpa/DataModel/AbstractData.php

```diff
@@ -120,9 +121,7 @@ public function set($property, $value)
             throw new InvalidArgumentException("$property is not a valid property");
         }
 
-        //  MongoDates should be converted to Datatime.
-        //  Once we have ext-mongo >= 1.6, we can use $value->toDateTime()
-        if (class_exists('\MongoDate') && $value instanceof \MongoDate) {
+        if (class_exists(MongoDate::class) && $value instanceof MongoDate) {
```

Comment:
I don't think you can use MongoDate::class here as it will cause an error if not present. Perhaps we should just remove the class_exists check?